warehouse/query-core/src/main/java/datawave/query/scheduler/Scheduler.java

```diff
@@ -1,37 +1,35 @@
 package datawave.query.scheduler;
 
-import java.util.Collection;
-import java.util.Map.Entry;
-
+import com.google.common.collect.Lists;
 import datawave.query.CloseableIterable;
 import datawave.query.config.ShardQueryConfiguration;
 import datawave.query.tables.ScannerFactory;
 import datawave.query.tables.stats.ScanSessionStats;
 import datawave.webservice.query.configuration.QueryData;
-
 import org.apache.accumulo.core.client.BatchScanner;
 import org.apache.accumulo.core.client.IteratorSetting;
 import org.apache.accumulo.core.client.TableNotFoundException;
-import org.apache.accumulo.core.data.Key;
-import org.apache.accumulo.core.data.Value;
 
-import com.google.common.collect.Lists;
+import java.util.Collection;
+
+/**
+ *
+ */
```

Comment:
Similar thing here, would like the generic parameter set in the annotation if there is a block for it. Otherwise removing the empty block could work too.

Reply:
I don't recall if I added the block with intent to fill it out or my IDE did it (I'd have to check settings), but I filled it out. Thanks